### PR TITLE
feat: export domain definitions types

### PR DESCRIPTION
**Pull request by Harasz**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -28,6 +31,9 @@ export {
   IRoleDefinitionV2,
   IRoleDefinitionText,
   PreconditionType,
+  IFieldDefinition,
+  IIssuerDefinition,
+  IRevokerDefinition,
```
Comment (JGiter) [inline src/index.ts:36]: Why not to reexport them?

Comment (Harasz) [reply src/index.ts:36]: What do you mean? I just fallowed previous convention with importing and then exporting.

Comment (JGiter) [reply src/index.ts:36]: Yes, I know it is not related to PR, but taking chance this file can be simplified. It is up to you though

**Review by JGiter — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (Harasz) [reply src/index.ts:36]: I will create a task for that :+1:

Comment (Harasz) [reply src/index.ts:36]: ICL-250

**Pull request merged**